website/content/docs/getting-started/deploy-and-login.mdx

```diff
@@ -88,34 +88,26 @@ instance, and click **Authenticate**.
    instance's Global scope via the web UI. This is the default scope for all new
    Boundary clusters.
 
-1. Click **Auth Methods**.
-
-1. Click the **copy icon** for the Password **Auth Method ID**. The ID is needed to
-authenticate to HCP Boundary using the CLI.
-
 <Tabs>
 <Tab heading="CLI">
 
-
 1. In the HCP Boundary portal, open the **Boundary Overview** page, and click
 the **copy icon** in the **Copy this into Boundary Desktop** section. This
 copies your environment’s Boundary **Cluster URL**.
 
 1. Open a terminal session and set the `BOUNDARY_ADDR` environment variable to
 the copied Cluster URL.
 
-```shell-session
-$ export BOUNDARY_ADDR=<boundary-cluster-url>
-```
+    ```shell-session
+    $ export BOUNDARY_ADDR=<boundary-cluster-url>
+    ```
```

Comment:
this is rendering incorrectly in the deploy preview, likely because of indentation

```suggestion
```shell-session
 $ export BOUNDARY_ADDR=<boundary-cluster-url>
```
```